### ISSUE TITLE
Profiler broken in 2.0

### ISSUE THREAD

**Issue by glensc** [propelorm/PropelBundle#301]
Key "options" for array with keys "adapter, user, password, dsn, classname, slaves" does not exist in PropelBundle:Panel:configuration.html.twig at line 51

i did clear cache (rm -rf), so that shouldn't be the cause

```
➔ composer show -i|grep propel
propel/propel                        dev-master 2302f06 Propel2 is an open-source Object-Relational Mapping (ORM) for PHP 5.4
propel/propel-bundle                 2.0.x-dev 3d74fbd  Integration of Propel in Symfony2
```

**Comment by jkabat** [comment]
@Czarnolecki exactly! 
Can this be fixed in 1.4 branch as well?

**Comment by marcj** [comment]
@havvg ?
